### DISCUSSION TITLE
How to build a 'yq' release binary from my cloned local repo

### DISCUSSION THREAD

**Discussion by TomTom876** [mikefarah/yq#1106] [Q&A]
yq is a great tool. My company don't allow to download a yq release binary from github. I will have to build it from source code. I wonder what is the command I can use to build the binary?

Thanks for your help!

**Comment by mikefarah** [comment]
Hi,

```bash
scripts/devtools.sh
make local build
```

That will run the build with all the linting and tests etc.

Otherwise, if you just want the binary:

```
go build .
```
